.github/workflows/lint.yml

```
@@ -0,0 +1,14 @@
+
+name: golangci-lint
+on:
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.37.1
```

If there's no reason to install an older one:

```suggestion
          version: v1.40.0
```